media-libs/libhdhomerun/libhdhomerun-20190621.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SiliconDust HDHomeRun Utilties"
+HOMEPAGE="https://www.silicondust.com/support/linux/"
+SRC_URI="https://download.silicondust.com/hdhomerun/${PN}_${PV}.tgz"
+
+PATCHES=(
+	"${FILESDIR}"/dont-strip.patch
+)
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
```

Comment:
It's up to you to make the final decision but I'd suggest you removed those empty variables. It looks weird that you assign an empty variable to an empty variable ;-).

Reply:
Good comment - removed